### PR TITLE
added code to bypass modal

### PR DESCRIPTION
**Pull request by tccharles**
#123 
Not sure if this is needed: https://github.com/mozilla/Affiliates-Tests/compare/master...tccharles:bypass_modal

### CODE REVIEW

**Comment by m8ttyB** [comment]
This is for issue #123 - @rbillings / @davehunt mind doing an r?

File: pages/start_page.py [hunk 1]
```diff
@@ -5,17 +5,21 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
 from selenium.webdriver.support.ui import WebDriverWait
 
 from pages.base import Base
 
-
```
Comment (davehunt) [inline pages/start_page.py]: We should restore this as it's a PEP8 rule. Top level definitions should be separated by two blank lines.

**Comment by davehunt** [comment]
It looks like this won't merge, so you may need to rebase from upstream. Other than my nit, this looks good to me. If you address that, I'd be happy to perform the rebase/merge for you.

**Comment by bensternthal** [comment]
Cursory glance looks good to me noting the merge issue, however I am not currently setup to run and verify this locally.

**Comment by m8ttyB** [comment]
Thanks @tccharles for the most excellent work and @davehunt for walking me through resolving merge conflicts. Closing this pr .. merged in https://github.com/mozilla/Affiliates-Tests/commit/2316843547c0a7246a0da063f3e985cedff07222

**Pull request closed without merging**